frontend/src/lib/components/DateFilter/DateFilterRange.tsx

```diff
@@ -3,7 +3,7 @@ import dayjs from 'dayjs'
 import { Button } from 'antd'
 
 import dayjsGenerateConfig from 'rc-picker/lib/generate/dayjs'
-import generatePicker from 'antd/es/date-picker/generatePicker'
+import generatePicker from 'antd/lib/date-picker/generatePicker'
```

Comment:
The antd documentation doesn't use one consistently either. [Their package.json](https://github.com/ant-design/ant-design/blob/master/package.json#L45) points to `antd/es/` in the module field which _supposedly_ helps bundlers (see https://github.com/rollup/rollup/wiki/pkg.module and https://webpack.js.org/guides/author-libraries/#final-steps)

It would be interesting to test if this has any impact on bundle speed or size... 

But I don't think we should block this PR on that - ideally, our tools should be clever enough to cope with us importing from any of the antd build folders so that we fall into the pit of success